stewards.json

```diff
@@ -10,5 +10,9 @@
     {
         "ca": "rockaway_staking/server_ca.crt",
         "endpoint": "https://steward-01.rbf.systems:5734"
+    },
+    {
+        "ca": "lavender_five/server_ca.crt",
+        "endpoint": "https://rpc.sommelier.circlewithadot.net:5734"
```

Comment:
This endpoint isn't accepting TCP connections on port 5734. Might be a firewall issue, config issue, or it may be down.

You'll also have to merge main and resolve the merge conflicts for this file.

Reply:
I checked and I do have port '5734' open on my firewall. I did see that the SSL cert I setup wasn't activated, maybe that was the issue? (it's since been activated)